agent/src/agent.ts

```
@@ -710,7 +712,10 @@ export class Agent extends MessageHandler {
         })
 
         this.registerAuthenticatedRequest('chat/restore', async ({ modelID, messages, chatID }) => {
-            const chatModel = new SimpleChatModel(modelID, [], chatID)
+            const chatModel =
+                modelID === null
+                    ? new SimpleChatModel(ModelProvider.get(ModelUsage.Chat).at(0)!.model, [], chatID)
```

It's always better to report a helpful error than to use `!`

```
const model = ModelProvider.get(ModelUsage.Chat).at(0)?.model
if (!model) { throw new Error( ... ) }
...
```